examples_replies.php

```diff
@@ -46,36 +46,48 @@
 // Create new MMSoap class
 $soap = new MMSoap($username, $password, $options);
 
-echo "** Send message to $recipient\n";
-// send the message
+echo "** Do you want to send a message to $recipient, y followed by ENTER to send, or ENTER to skip.\n";
+$handle = fopen("php://stdin", "r");
+$line = fgets($handle);
+if ($line == "y\n") {
+    // send the message
 
-$response = $soap->sendMessage ( $recipient, "messagemedia-php: please reply" );
-if ($response instanceof SoapFault) {
-	exit ( 'Error: ' . $response->getMessage () );
-}
-$result = $response->getResult ();
-echo $result->sent . ' sent / ' . $result->scheduled . ' scheduled / ' . $result->failed . " failed\n\n";
+    $response = $soap->sendMessage($recipient, "messagemedia-php: please reply & press Enter on example when done.");
+    if ($response instanceof SoapFault) {
+        exit ( 'Error: ' . $response->getMessage());
+    }
+    $result = $response->getResult ();
+    echo $result->sent . ' sent / ' . $result->scheduled . ' scheduled / ' . $result->failed . " failed\n\n";
 
-// message has been sent, give the user time to reply
-echo "Please reply to the message you just received and hit ENTER.\n";
-$handle = fopen ( "php://stdin", "r" );
-$line = fgets ( $handle );
+    // message has been sent, give the user time to reply
+    echo "Please reply to the message you just received and hit ENTER.\n";
+    $handle = fopen("php://stdin", "r");
```

Comment:
Could these two lines be extracted to a function of their own? This is the second use of this exact pattern?

Maybe, $input = getUserInput()?